src/sentry/notifications/utils/__init__.py

```diff
@@ -439,11 +455,15 @@ def generic_email_html(context: Any) -> Any:
 
 def get_performance_issue_alert_subtitle(event: Event) -> str:
     """Generate the issue alert subtitle for performance issues"""
-    spans, matched_problem = get_span_and_problem(event)
     repeating_span_value = ""
-    if spans and matched_problem:
-        _, repeating_spans = get_parent_and_repeating_spans(spans, matched_problem)
-        repeating_span_value = get_span_evidence_value(repeating_spans, include_op=False)
+    if isinstance(event, GroupEvent) and event.occurrence is not None:
+        repeating_span_value = event.occurrence.evidence_data["repeating_spans_compact"][0]
```

Comment:
Might be a good idea to check that this value actually exists, just in case someone messes up a detector in the future?

Reply:
done!